meta-xt-prod-devel-rcar-control/recipes-guests/domd/files/domd-salvator-xs-h3-4x2g.cfg

```diff
@@ -14,7 +14,7 @@ device_tree = "/usr/lib/xen/boot/domd.dtb"
 
 # Kernel command line options
 # Uncomment this when using nfs as a boot device
-#extra = "root=/dev/nfs nfsroot=192.168.1.100:/srv/domd,vers=3 ip=dhcp rw rootwait console=hvc0 pvrsrvkm.DriverMode=0 xt_page_pool=67108864 xt_cma=100663296"
+#extra = "video=VGA-1:d video=LVDS-1:d root=/dev/nfs nfsroot=192.168.1.100:/srv/domd,vers=3 ip=dhcp rw rootwait console=hvc0 pvrsrvkm.DriverMode=0 xt_page_pool=67108864 xt_cma=100663296"
```

Comment:
I believe, this us not used anymore, as we use `set root script` now. So, this commented  `extra` can be removed once and for all

Reply:
makes sense, will remove

Reply:
"+"